lesson_10/libraries/src/loaders/loaders.module.ts

```diff
@@ -1,10 +1,10 @@
 import { Module } from '@nestjs/common';
 import { AnthonyMaysLoader } from './anthony_mays_loader.js';
-
+import { XavierCruzLoader } from './xavier_cruz_loader.js';
 export const Loaders = Symbol.for('Loaders');
```

Comment:
Should be a space before this line, otherwise this looks good!

Reply:
Should be done!